### PR TITLE
tweaks

### PR DESCRIPTION
**Pull request by mhupman**

### CODE REVIEW

File: soyjs/exec.go [hunk 1]
```diff
@@ -406,6 +396,9 @@ func (s *state) visitPrint(node *ast.PrintNode) {
 			// no implementation, they just serve as a marker to cancel autoescape.
 		default:
```
Comment (ovkulkarni) [inline soyjs/exec.go:397]: Will our custom directives get added into soyjs.PrintDirectives? If not, it will never get to this default case.

**Comment by ovkulkarni** [comment]
I also just tested this locally with an unmodified version of soy2js (no import rewriting), and it seems like I get quite a few of these: `import {  } from '.js';`. Additionally, this creates import statements for things like `JSON.stringify` -> `import { JSON__stringify } from 'JSON.stringify.js';`

File: soyjs/exec.go (hunk 1 above)
Comment (mhupman) [reply soyjs/exec.go:397]: Yes, all custom directives/funcs must be added to soyjs.Directives/soyjs.Funcs, respectively

**Comment by mhupman** [comment]
Ah, thanks for the heads up!  Will look now....

**Pull request merged**